### ISSUE TITLE
Home Assistant 0.87.0: debug function fails to find and set title

### ISSUE THREAD

**Issue by pohly** [thomasloven/lovelace-player#1]
I'm trying your revision 43020cb (current master) with Home Assistant 0.87.0, but get some JavaScript errors. The `debug` function failed (visible in the Firefox developer console) because the element lookups in https://github.com/thomasloven/lovelace-player/blob/43020cbaccd50d8a7d14520b60e8cdf0ac39992e/lovelace-player.js#L98-L106 led to a nil value.

More specifically, Chrome points to line 103, the result of the `querySelector("#panel ha-panel-lovelace")`:
```
lovelace-player.js:103 Uncaught TypeError: Cannot read property 'shadowRoot' of null
    at Object.debug (lovelace-player.js:103)
    at lovelace-player-config.js?v=4:8
```

**Comment by pohly** [comment]
My workaround is this:
```
diff --git a/lovelace-player.js b/lovelace-player.js
index 61d2832..b6b20e2 100644
--- a/lovelace-player.js
+++ b/lovelace-player.js
@@ -94,7 +94,8 @@ var LovelacePlayer = LovelacePlayer || (function() {
     debug: () => {
       let message = "";
       _players.forEach(p => { message = message + " " + p; });
       if (message === "") message = "NONE";
+      window.alert("LovelacePlayer Device ID: " +_getDeviceId() + " Bound to: " + message)
       document
         .querySelector("home-assistant")
         .shadowRoot.querySelector("home-assistant-main")
```

**Comment by pohly** [comment]
That now shows that my browser is bound to my new `media_player.browser` entity. Sending `{"message": "I am speaking now"}` to the `tts.google_say` service does end up calling the kiosk player:
```
Feb 10 20:59:43 xbmc hass[3163]: 2019-02-10 20:59:43 WARNING (MainThread) [homeassistant.helpers.service] Not passing an entity ID to a service to target all entities is deprecated. Update your call to media_player.play_media to be instead: entity_id: all
Feb 10 20:59:43 xbmc hass[3163]: 2019-02-10 20:59:43 INFO (SyncWorker_17) [custom_components.media_player.floorplan_speaker] play_media: http://192.168.1.168:8123/api/tts_proxy/ace22c85e1db73d4c89012bf1ed2635a333faa26_en_-_google.mp3
```

I can retrieve the mp3 file and play it. But `lovelace-player` remains silent.

Further investigation shows that testing this way causes `lovelace-player` to ignore the play event because it doesn't have an entity id. My workaround:
```
index 61d2832..f488d1b 100644
--- a/lovelace-player.js
+++ b/lovelace-player.js
@@ -44,7 +44,7 @@ var LovelacePlayer = LovelacePlayer || (function() {
       let targets = event.data.service_data.entity_id;
       if(!Array.isArray(targets)) { targets = Array(targets); }
       let istarget = _players.some(r => targets.indexOf(r) >= 0);
-      if(!istarget) { return; }
+      if(!istarget && event.data.service_data.entity_id) { return; }
       switch(event.data.service) {
         case 'play_media':
           _play(event.data.service_data.media_content_id);
```

But still nothing. I can see in a debugger that `_play` gets called with the right URL, but Firefox 60.4.0esr remains silent.

I wonder if this might be related to efforts to silence noisy web sites? I don't know how the player works under the hood, but as it starts playing without an obvious user input, it's not different from other web sites where recent(lish) browsers got more aggressive about suppressing media playback. Just a thought, I haven't really followed that area that much.

**Comment by pohly** [comment]
With the Chrome browser on an old Android phone I got playback to work, which is actually what I wanted to achieve. Thanks for `lovelace-player`, it turned out to be exactly what I needed.

However, the title isn't shown on the Android phone either. It was visible before reloading the page, perhaps because then some parts of the page still came from the older Home Assistant.

**Comment by jazzyisj** [comment]
I can confirm that on Hassio 89.2 with commit  43020cb using Chrome 73.0.3683.86 (Official Build) (64-bit)  the device ID does not appear in the title.  I have no idea how to find it on my own, so I am unable to test or use this component.

I have restarted Hassio, deleted my browsing history, done hard reset/empty cache in dev mode, and created a new ui-lovelace.yaml with lovelace gen.

Edit:  I have found the following entry in my HA log.

`ERROR (MainThread) [frontend.js.latest.201903051] https://<redacted>:8123/local/lovelace-player.js:103:9 Uncaught TypeError: Cannot read property 'shadowRoot' of null`
